tests/keys/rsa256/private.pem

```diff
@@ -0,0 +1,15 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
If this is not a secret can you add a comment to the top of the file, so it doesn't scare people.

Reply:
Added